### PR TITLE
download: add links for arm darwin binaries

### PR DESCRIPTION
**Pull request by katesalazar**
This shall close ticket #917.

### CODE REVIEW

File: _includes/templates/download.html [hunk 1]
```diff
@@ -48,11 +48,19 @@ <h2>{{ page.latestversion }} {{CURRENT_RELEASE}} <a type="application/rss+xml" h
...
         <img src="/assets/images/os/med_osx.png" alt="osx">
         <span>
-          <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.macdmg }}">macOS</a>
+          <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.macdmg }}">macOS (x86_64)</a>
           <span><a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.macdmg }}" class="dl" id="macdmg">dmg</a> -
             <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.mactar }}" class="dl" id="mactar">tar.gz</a></span>
         </span>
       </div>
+      <div>
+        <img src="/assets/images/os/med_osx.png" alt="osx" style="visibility:hidden">
```
Comment (fanquake) [inline _includes/templates/download.html:57]: Why `visibility:hidden`?

**Review by fanquake — COMMENTED** [review]
Can you provide a screenshot of what this actually looks like.

File: _includes/templates/download.html (hunk 1 above)
Comment (katesalazar) [reply _includes/templates/download.html:57]: Intended to make invisible the image while having the image occupy the same block of space.

Although DOM-untidy, this tends to render OK-ish.

**Comment by katesalazar** [comment]
Screen Shot 2022-11-15 at 10 56 42
Screen Shot 2022-11-15 at 10 56 24

File: _includes/templates/download.html (hunk 1 above)
Comment (MarcoFalke) [reply _includes/templates/download.html:57]: Agree  it should be visible if it takes up another square

Comment (katesalazar) [reply _includes/templates/download.html]: force pushed a rev, same contents, just removing the visibility attr

**Comment by fanquake** [comment]
Can you provide a new screenshot. Either by updating your previous comment, or deleting it and posting a new one.

**Comment by katesalazar** [comment]
screenshots using d83eeeb77ceaef62a196e5263177b51ca13e2971
my normal resolution:
Screen Shot 2022-11-15 at 14 28 08
if I maximize:
Screen Shot 2022-11-15 at 14 28 21

**Review by fanquake — APPROVED** [review]
I think this looks fine now. Haven't tested.

**Comment by hebasto** [comment]
Should this snippet https://github.com/bitcoin-core/bitcoincore.org/blob/d83eeeb77ceaef62a196e5263177b51ca13e2971/_includes/templates/download.html#L321-L324 be also modified to assign the correct file to the Download button when browsing on macOS arm64?

**Comment by katesalazar** [comment]
ah, the tricky bit

I can push another rev with some swiftly changes,
but I don't have the devices to check if it'll be working

**Comment by fanquake** [comment]
There's no need to bundle that change into this PR. It can be done separately as part of #696.

**Comment by katesalazar** [comment]
I think a68a7e89b76 should do, but I have not the hardware to check

if does look good write something and I squash asap

File: _includes/templates/download.html [hunk 2]
```diff
@@ -295,7 +295,12 @@ <h2 style="text-align: center">{{page.build_reproduction}}</h2>
 
 <script type="text/javascript">
 var os = 'windows64';
-if (navigator.userAgent.indexOf('Mac') != -1) var os = 'mac'
+if (navigator.userAgent.indexOf('Mac') != -1) {
+	var os = 'mac';
+	if (navigator.userAgent.indexOf('Intel') == -1) {
```
Comment (katesalazar) [inline _includes/templates/download.html:300]: yeah... I'm reading stuff that says this doesn't work

Comment (katesalazar) [reply _includes/templates/download.html:300]: at least this rev doesn't break anything for the intel macs

**Comment by fanquake** [comment]
> There's no need to bundle that change into this PR. It can be done separately as part of https://github.com/bitcoin-core/bitcoincore.org/issues/696.

Can you just drop the untested changes.

**Comment by katesalazar** [comment]
> > There's no need to bundle that change into this PR. It can be done separately as part of #696.
> 
> Can you just drop the untested changes.

Yeah, done!

**Pull request merged**